src/aws_encryption_sdk/__init__.py

```diff
@@ -36,6 +35,8 @@
     StreamEncryptor,
 )
 
+check_python_version()
```

Comment:
@farleyb-amazon I do not know how to force a Runtime Check, but I think this will work.
Do you know of another way? 
I fear that this could run multiple times in an application if they load the library multiple times; but I guess I am willing to allow that to happen.
I also assume that `__init__` will ALWAYS be parsed if any part of this library is used. I think that assumption is accurate... but I cannot swear it.